### ISSUE TITLE
mingw C api release does not contain an import library for the shared library

### ISSUE THREAD

**Issue by madebr** [bytecodealliance/wasmtime#3168]
The 0.29.0 release does not contain an import library for the shared library.
It should contain a file named `lib/libwasmtime.dll.a`.

The contents of `wasmtime-v0.29.0-x86_64-mingw-c-api.zip` are:
```
wasmtime-v0.29.0-x86_64-mingw-c-api
├── include
│   ├── doc-wasm.h
│   ├── wasi.h
│   ├── wasm.h
│   ├── wasmtime
│   │   ├── config.h
│   │   ├── error.h
│   │   ├── extern.h
│   │   ├── func.h
│   │   ├── global.h
│   │   ├── instance.h
│   │   ├── linker.h
│   │   ├── memory.h
│   │   ├── module.h
│   │   ├── store.h
│   │   ├── table.h
│   │   ├── trap.h
│   │   └── val.h
│   └── wasmtime.h
├── lib
│   ├── libwasmtime.a
│   └── wasmtime.dll
├── LICENSE
└── README.md

3 directories, 21 files
```

**Comment by bjorn3** [comment]
Rustc doesn't create import libraries afaik. You will need to make them yourself. According to [stackoverflow](https://stackoverflow.com/questions/9946322/how-to-generate-an-import-library-lib-file-from-a-dll) you can use the following commands:

```shell
echo LIBRARY WASMTIME > wasmtime.def
echo EXPORTS >> wasmtime.def
for /f "skip=19 tokens=4" %A in ('dumpbin /exports wasmtime.dll') do echo %A >> wasmtime.def
lib /def:wasmtime.def /out:wasmtime.lib
```

**Comment by alexcrichton** [comment]
Thanks for the report! This was forgotten by accident, and I've submitted https://github.com/bytecodealliance/wasmtime/pull/3170 to fix this.

**Comment by pepyakin** [comment]
Closed by #3170
